llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -1974,7 +1974,18 @@ define void @foo(i32 %arg) {
   EXPECT_EQ(PHI->hasConstantOrUndefValue(), LLVMPHI->hasConstantOrUndefValue());
   // Check isComplete().
   EXPECT_EQ(PHI->isComplete(), LLVMPHI->isComplete());
-
+  // Check replaceIncomingBlockWith
+  OrigBB = PHI->getIncomingBlock(0);
+  EXPECT_EQ(OrigBB, BB1);
```

Comment:
nit: I would use a `NewBB` variable to make it a bit more explicit, and use `OrigBB` and `NewBB` in `replaceIncomingBlockWith()`, like so:
```
OrigBB = PHI->getIncomingBlock(0);
auto *NewBB = BB2;
EXPECT_NE(NewBB, OrigBB);
PHI->replaceIncomingBlockWith(OrigBB, NewBB);
```